@stellar/design-system-website/src/generated/gitInfo.ts

```diff
@@ -1 +1 @@
-export default {"commitHash": "b7e4319"};
+export default {"commitHash": "095eacc"};
```

Comment:
❔ What is this used for?

Reply:
It's just a commit hash used on the website to check which version was built. At one time we had an issue with build and it was hard to investigate not knowing which is the actual code we're looking at.

![image](https://user-images.githubusercontent.com/7346473/148611973-9f43d229-12de-4e31-86c4-f80f9d4a33f6.png)